index.js

```diff
@@ -14,14 +21,43 @@ function sortByStringLength() {}
  * Example: "Hello"
  * [ 'elloH', 'lloHe', 'loHel', 'oHell', 'Hello' ]
  */
-function textScroller() {}
+function textScroller(word) {
+  console.log(word)
+  let newArrayOfStrings = [];
+  for (let i = 0; i < word.length; i++) {
+    newArrayOfStrings.unshift(word.slice(word.length - i) + word.slice(0, word.length - i));
+  }
+  console.log(newArrayOfStrings)
+  return newArrayOfStrings;
+}
+  
+
+
 
 /**
  * Returns the difference between the largest and smallest number in the array
  * @param {Number[]} numbers - An array of numbers.
  * @returns {Number} The difference between the largest and smallest number.
  */
-function betweenExtremes() {}
+function betweenExtremes(numbers) {
+  if(numbers.length == 0) {
+    return [];
+  }
+
+  if(!numbers.every(number => typeof(number) == "number")){
+    return numbers;
+  }
+
+  numbers = numbers.sort((a,b) => a - b);
```

Comment:
remember `.sort()` mutates the array.  This is the same as `numbers.sort((a,b) => a - b);`